src-docs/src/views/elastic_charts/accessibility_example.js

```diff
@@ -0,0 +1,313 @@
+            <EuiCode>{'<Settings />'}</EuiCode> component.
+            <EuiCode>{'ariaDescription'}</EuiCode> takes a string description
+            which will be visually hidden whereas{' '}
+            <EuiCode>{'ariaDescribedBy'}</EuiCode> prop takes an{' '}
+            <EuiCode>{'id'}</EuiCode> that you have rendered elsewhere in your
+            app.
+          </p>
+          <EuiCodeBlock language="jsx" isCopyable fontSize="s">
+            {`<Chart>
+  <Settings
+    ariaLabel="Days of the week: weekdays vs weekends"
+    ariaDescription="Bar one is labelled weekdays and has a value of 5. Bar two is labelled weekends and has a value of 2."
+  />
+  <BarSeries {...props} />
+</Chart>`}
+          </EuiCodeBlock>
+          <EuiCallOut
+            title={
+              <>
+                <EuiCode>{'ariaDescribedBy'}</EuiCode> will override{' '}
+                <EuiCode>{'ariaDescription'}</EuiCode> if both are specified.
+              </>
+            }
+            color="warning"
+            iconType="alert"
+          />
+          <EuiSpacer />
+          <p>
+            <EuiCode>{'ariaUseDefaultSummary'}</EuiCode> is a boolean defaulted
+            to <EuiCode>{'true'}</EuiCode>. The default summary is bare bones
+            and only reports the chart type(s). Some chart types provide more
+            information but today that is only Gauge and Goal charts. If you are
+            passing in a custom description, you can disable this prop.
+          </p>
+          <EuiCodeBlock language="jsx" isCopyable fontSize="s">
+            {`<Fragment>
+  <h1 id="randomHeadingId">Days of the week: weekdays vs weekends</h1>
+  <p id="randomDescId">Bar chart with two bars. Bar one is labelled weekdays and has a value of 5. Bar two is labelled weekends and has a value of 2.</p>
+  <Chart>
+    <Settings
+      ariaLabelledby="randomHeadingId"
+      ariaDescribedBy="randomDescId"
+      ariaUseDefaultSummary={false}
+    />
+    <BarSeries {...props} />
+  </Chart>
+</Fragment>`}
+          </EuiCodeBlock>
+        </>
+      ),
+    },
+    {
+      title: 'Tables',
+      text: (
+        <>
+          <p>
+            The <EuiCode>{'ariaTableCaption'}</EuiCode> prop is available to set
+            a caption to the data table for users of assistive technologies. The
+            below table is an example autogenerated for a sunburst chart. If
+            there are many data points, the data table will only generate some
+            of the data each time to avoid overwhelming users.
+          </p>
+          <EuiCallOut
+            title={
+              <>
+                Currently, data tables are only available for{' '}
+                <strong>partition charts</strong>.
+              </>
+            }
+            iconType="visPie">
+            <p>
+              At time of this writing, partition charts include: sunburst,
+              treemap, icicle, flame, and mosaic.
+            </p>
+          </EuiCallOut>
```

Comment:
I followed a consistent format and location for where to mention that specific things only work on specific charts